src/main/resources/jenkins/plugins/slack/user/SlackUserProperty/config.jelly

```diff
@@ -0,0 +1,9 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry title="Slack User Id">
```

Comment:
I know this plugin hasn't done i18n well but for new code we should make it i18n aware
```suggestion
    <f:entry title="${%Slack User Id}">
```